idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -124,6 +124,10 @@ def fq2fa(input_fastq, output_fasta):
     execute_command(cmd)
 
 
+def result_path(basename):
```

Comment:
There is (or was?) a similar result_dir function in non_host_alignment_functions.py, if I recall correctly.

Reply:
Ah ok I changed result_dir to be called result_path but I think I'll leave the declarations separate since common.py doesn't get the RESULT_DIR (SAMPLE_DIR) declarations

Reply:
Sounds good.